mysql-test/suite/rpl/t/bug80821.test

```diff
@@ -0,0 +1,45 @@
+--source include/not_ndb_default.inc
+--source include/not_group_replication_plugin.inc
```

Comment:
Why?

Reply:
I've used the testcase from the jira ticket, and these were present in it. I didn't know why, so I kept them. I'll remove them.